api/queries_pr.go

```diff
@@ -261,15 +261,23 @@ type PullRequestChecksStatus struct {
 	Total   int
 }
 
-func (pr *PullRequest) ChecksStatus() (summary PullRequestChecksStatus) {
+func (pr *PullRequest) ChecksStatus() PullRequestChecksStatus {
```

Comment:
I feel fairly strongly about not using named returns in the usual case for reasons well described here: https://dave.cheney.net/practical-go/presentations/gophercon-israel.html#_avoid_named_return_values